motr/setup.c

```diff
@@ -2291,8 +2292,8 @@ static int _args_parse(struct m0_motr *cctx, int argc, char **argv)
 				LAMBDA(void, (const char *s)
 				{
                                         char tmp_buf[512];
-                                        sprintf(tmp_buf, "%s-%d", s, (int)m0_pid());
-                                        rctx->rc_addb_stlocation = strdup(tmp_buf);
+                                        snprintf(tmp_buf,sizeof(tmp_buf) ,"%s-%d", s, (int)m0_pid());
+                                        rctx->rc_addb_stlocation = m0_strdup(tmp_buf);
```

Comment:
Please limit the line to 80 chars.

Reply:
And take care of the spaces between identifiers.

Reply:
Updated code with proper spacing and total number of character in the line are 63.

Reply:
In my text editor, it is 102 chars.
Can you please check?